src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -247,7 +247,7 @@ public IExternalControl Control
 
         // Stores tokens that need replacement (Display Name -> Logical Name) for serialization
         // Replace Nodes (Display Name -> Logical Name) for serialization
-        public IList<KeyValuePair<Token, string>> NodesToReplace { get; }
+        public IDictionary<Token, string> NodesToReplace { get; }
```

Comment:
>IDictionary [](http://example.com/codeflow?start=15&length=11)

This ensures that conflicts don't crash  - but the fundamental question is *why* are we getting conflicts?

Reply:
With an expression like "SortByColumns(Accounts, 'Account Name', SortOrder.Ascending)" (coming from TestDelegableExpressions_PowerFxV1 test), 'Account Name' second argument is processed in Binder.cs
- first at line 4574, which ends up in calling GetLogicalNodeNameAndUpdateDisplayNames at line 2813
- then, a second time at line 4616 where we call GetLogicalNodeNameAndUpdateDisplayNames again
GetLogicalNodeNameAndUpdateDisplayNames ends up at line 2735 adding the same token (with the same span and string) twice.

The code in ExpressionLocalizationHelper.ConvertExpression uses an algorithm based on a worklist which is a Dictionary and requires unicity of Token's spans.
This fix is simply removing the duplicate spans (that always contain the exact same Token/string couple as this is the same input expression).